test/test_camera_properties.py

```diff
@@ -1,8 +1,7 @@
 """
 Tests for the camera_properties class
 """
-from camera_properties import CameraProperties
-
+from modules.vision.module.camera_properties import CameraProperties
 
```

Comment:
Missing blank line according to pep8

Reply:
Fixed in https://github.com/R2D2-2019/vision/pull/12/commits/3d85478d0dad06c1d22054b8fbdc58f652e17c44